frappe/public/js/frappe/form/form.js

```diff
@@ -1707,7 +1707,7 @@ frappe.ui.form.Form = class FrappeForm {
 		for(var i = 0; i < cl.length; i++){
 			if(!cl[i][fieldname]) cl[i][fieldname] = value;
 		}
-		refresh_field("items");
+		this.refresh_field("items");
```

Comment:
- Shouldn't this be the child table name instead of items here, aren't we updating the `fieldvalue` for `table_fieldname`? If not, am I missing something here ?
- Also could you change the comment, its feels misleading.

Reply:
You're right, I was too focused on the fact that it calls the refresh function in a complicated way. :D

Refactored the whole thing now. It doesn't even need the refresh anymore.